smithy-linters/src/test/resources/software/amazon/smithy/linters/errorfiles/deprecated-traits-validator.json

```diff
@@ -0,0 +1,29 @@
+{
+  "smithy": "0.5.0",
+  "shapes": {
+    "ns.foo#MyString": {
+      "type": "string",
+      "traits": {
+        "ns.foo#myTraitName": { }
+      }
+    },
+    "ns.foo#myTraitName": {
+      "type": "structure",
+      "members": { },
+      "traits": {
+        "smithy.api#deprecated": { },
+        "smithy.api#trait": {
+          "selector": "*"
+        }
+      }
+    }
+  },
+  "metadata": {
+    "validators": [
+      {
+        "id": "Deprecated Traits",
+        "name": "DeprecatedTraits"
```

Comment:
This isn't needed. The DeprecatedTraits validator should always run.

Reply:
Fixed in 8abdbc17

Dropped the ID, but the name is used in the test runner.